src/utils/proxy.js

```diff
@@ -35,7 +39,7 @@ async function getStatic(pathname, publicFolder) {
   for (const i in alternatives) {
     const p = alternatives[i]
     try {
-      const pathStats = await fs.stat(p)
+      const pathStats = await fsStatAsync(p)
```

Comment:
This is the only async call to fs.* APIs in this file, it could just become `fs.statSync`, as this should be pretty fast, or we could just convert all other calls to use their async alternative.

Reply:
How about using the sync method and opening a new issue to refactor this file to use `async` functions?

Reply:
Yep, I will open one as soon as I get a chance 👍

Reply:
done: https://github.com/netlify/cli/issues/1275